lib/build.ts

```diff
@@ -166,6 +166,26 @@ async function compile(nodeVersion: string, targetArch: string) {
   return compileOnUnix(nodeVersion, targetArch);
 }
 
+async function hash(filePath: string): Promise<string> {
+  return new Promise<string>((resolve, reject) => {
+    const resultHash = crypto.createHash('sha256');
+    const input = fs.createReadStream(filePath);
+
+    input.on('error', (e) => {
+      reject(e);
+    });
+
+    input.on('readable', () => {
+      const data = input.read();
+      if (data) {
+        resultHash.update(data);
+      } else {
+        resolve(resultHash.digest('hex'));
```

Comment:
Should we also destroy the stream here?

Reply:
There is no need to. The process exits shortly after anyways.